colaraz/lms/static/js/discussion/colaraz_discussion_utils.js

```diff
@@ -0,0 +1,57 @@
+$(document).on("click", "button.action-vote", function(event) {
+    var currentElement = this;
+    if ($(currentElement).closest("article.discussion-article").length <= 0) {
+      return;
+    }
+
+    var is_voting_disabled = $(currentElement)
```

Comment:
```suggestion
    var is_voting_disabled = $(currentElement)
```
```suggestion
    var isVotingDisabled = $(currentElement)
```
please use camelCase notation in your variable names